lib/src/models/interactive_message/ds_interactive_message_header.model.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:get/get.dart';
+
+import '../../enums/ds_interactive_message_header_type.enum.dart';
+import 'ds_interactive_message_document.model.dart';
+import 'ds_interactive_message_image.model.dart';
+import 'ds_interactive_message_video.model.dart';
+
+class DSInteractiveMessageHeader {
+  final DSInteractiveMessageHeaderType? type;
+  final String? text;
+  final DSInteractiveMessageDocument? document;
+  final DSInteractiveMessageImage? image;
+  final DSInteractiveMessageVideo? video;
```

Comment:
Since this `DSInteractiveMessageDocument`, `DSInteractiveMessageDocument` and `DSInteractiveMessageVideo` has the same structure, I recommend you create a base class and extends it to create other class.

like as `DSInteractiveMessageMedia`

what do you think about?

Reply:
Done! 👍🏻